runtime/CMakeLists.txt

```diff
@@ -35,7 +36,9 @@ set(SHARED_RUNTIME_SOURCES
   ${CMAKE_CURRENT_SOURCE_DIR}/Shadow.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/GarbageCollection.cpp)
 
-if (${QSYM_BACKEND})
+if (${RUST_BACKEND})
```

Comment:
Minor: I think it would make sense to rename the backend in a way that prevents confusion with a possible future backend for _analyzing_ Rust programs (as opposed to your approach of enabling a backend implemented in Rust).